corehq/toggles.py

```diff
@@ -872,6 +872,13 @@ def _enable_search_index(domain, enabled):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
+MOBILE_UCR_TILE = StaticToggle(
```

Comment:
Is it necessary to have a separate toggle? Why not put this under the existing `MOBILE_UCR` toggle?

Reply:
sounds reasonable, 39d43e31d7b

Reply:
sounds reasonable, 39d43e31d7b